static/app/views/acceptOrganizationInvite/index.tsx

```diff
@@ -287,6 +284,7 @@ class AcceptOrganizationInvite extends DeprecatedAsyncView<Props, State> {
 
     return (
       <NarrowLayout>
+        <SentryDocumentTitle title={t('Accept Organization Invite')} />
```

Comment:
Anywhere where this doesn't wrap is because it's already a leaf view, so there would be no inner document titles where thee wrapping is required.